client/src/components/embedding/index.tsx

```diff
@@ -188,8 +77,7 @@ class Embedding extends React.PureComponent<{}, EmbeddingState> {
               <Button
                 type="button"
                 data-testid="layout-choice"
-                icon={heatmapIcon}
-                // minimal
+                icon={<SvgIcon src={heatmapIcon} />}
```

Comment:
here we render SvgIcon component and pass the SVG file path to it (processed by `file-loader`)